Solutions/moritz_w/Lesson_6/integer_sort.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+
+def insertion_sort(data):
+    sorted_data = [i for i in data]
+    for i in range(1, len(sorted_data)):
+        num = sorted_data[i]
```

Comment:
In python there is nice way of swapping variables without temp variables:
a, b = b, a
this will swap contents a with b and b with a.